docs-internal/api/nu-designer-openapi.yaml

```diff
@@ -1480,6 +1482,12 @@ components:
     AdditionalInfo:
       oneOf:
       - $ref: '#/components/schemas/MarkdownAdditionalInfo'
+    All:
```

Comment:
It shows that Schema for `allowedProcessingModes` is wrong. I saw that we encode it as an array of processing modes. Here it looks like we should provide `AllowedProcessingModes` object